bin/version

```diff
@@ -21,4 +21,3 @@ sed -i '' 's/\(\@iabtcf\/.*\)":.*"[0-9]*\..*"/\1": "'$npm_package_version'"/g' .
 # add this to the previous commit
 git add -A && git commit --amend --no-edit
 
-./bin/publish-version
```

Comment:
removing this to be able to call `npm run postversion` to update all packages to the desired version without the script trying to publish to npm, as we do this manually